src/main/java/com/bookstore/bookstoreapi/data/controller/AuthorController.java

```diff
@@ -0,0 +1,28 @@
+package com.bookstore.bookstoreapi.data.controller;
+
+import com.bookstore.bookstoreapi.data.entity.Author;
+import com.bookstore.bookstoreapi.data.service.AuthorService;
+import java.util.Optional;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/authors")
+public class AuthorController {
+
+  @Autowired private AuthorService authorService;
+
+  @PostMapping("/create")
+  public ResponseEntity<Author> createAuthor(@RequestBody Author author) {
+    Optional<Author> createdAuthor = authorService.createAuthor(author);
```

Comment:
Please always add empty line after the start of a method, also return should be isolated the same way by having empty line before it.